package.json

```diff
@@ -5,15 +5,20 @@
   "repository": "https://github.com/choonkending/react-webpack-node",
   "main": "index.js",
   "scripts": {
-    "start": "node server/index.js",
-    "build": "webpack --progress",
-    "watch": "webpack --watch --progress",
+    "start": "NODE_ENV=production node server/index.js",
+    "dev": "NODE_ENV=development node server/index.js",
+    "build": "webpack --progress --colors --config ./webpack/webpack.config.js",
+    "buildDev": "webpack --progress --colors --config ./webpack/webpack-hot-dev-server.config.js",
```

Comment:
Do you need to add a ```--watch``` here? its the only way I can get it to work correctly. Other wise some of the data isn't getting reloaded - for example props on a component.